config/routes/redirects.rb

```diff
@@ -3,4 +3,5 @@
   get '/courses/web-development-101(/lessons/:id)', to: redirect('/paths/foundations')
   get '/paths/:path_id/courses/:course_id/lessons/:id', to: redirect(Redirectors::NestedLessonRedirector.new)
   get '/discord', to: redirect(ODIN_CHAT_URL)
+  get '/blog', to: redirect(ODIN_BLOG_URL)
```

Comment:
It would be worth giving this a named route helper. That would allow us to change the path in the future (unlikely but you never know) here without needing to update the places where it's used.

```suggestion
  get '/blog', to: redirect(ODIN_BLOG_URL), as: :blog
```

Not related to your change - but it would be worth doing it for the discord route above this too if you'd be for doing that?